.travis.yml

```diff
@@ -9,11 +9,14 @@ cache:
     - node_modules
 
 before_install:
+  - sudo apt-key adv --fetch-keys http://dl.yarnpkg.com/debian/pubkey.gpg
```

Comment:
Thanks for the PR! For posterity, is the official reference you were noting this one? https://yarnpkg.com/lang/en/docs/install-ci/

Reply:
Exactly, it is https://yarnpkg.com/lang/en/docs/install-ci/#travis

I should have mentioned this.